MagisterkaApp.Calculator/ReadFile.cs

```diff
@@ -155,17 +167,24 @@ public static ObservableCollection<FrequencyStep> ReadMonitoringFile(List<string
                     int frequencyStepsCounter = 0;
 
                     while ((line = file.ReadLine()) != null)
-                    {                        
+                    {
                         if (counter == 0 && line.Any(x => char.IsLetterOrDigit(x)))
                         {
                             filtredLine = line.Replace(".", ",");
                             resultLine = filtredLine.Split(new char[] { ' ' }).ToArray();
 
                             if (resultLine.Length < 2)
-                                break ;
+                                break;
 
-                            var frequency = double.Parse(resultLine[frequencePosition]);
-                            var power = double.Parse(resultLine[powerPosition]);
+                            var separator = CultureInfo.CurrentCulture.NumberFormat.NumberDecimalSeparator;
+                            // Replace any periods or commas with the current culture separator and parse
+
+
+                            var frequency = double.Parse(Regex.Replace(resultLine[frequencePosition], "[.,]", separator));
```

Comment:
Same. We don't need to change the reading of numbers from point to comma for Windows 10. This change is only for Windows 7.